gremlin-python/src/main/python/gremlin_python/driver/client.py

```diff
@@ -55,8 +56,11 @@ def __init__(self, url, traversal_source, protocol_factory=None,
         self._headers = headers
         self._enable_user_agent_on_connect = enable_user_agent_on_connect
         self._traversal_source = traversal_source
+        self._enable_compression = enable_compression
         if not self._use_http and "max_content_length" not in transport_kwargs:
             transport_kwargs["max_content_length"] = 10 * 1024 * 1024
+        if self._enable_compression and "compress" not in transport_kwargs:
+            transport_kwargs["compress"] = 15  # enable per-message deflate compression with max 32k sliding window size
```

Comment:
This seems oddly specific. Is it an AioHttp value? It probably shouldn't work like this since this should work for any "transport".

Reply:
Yes it's an AioHttp value. I'll make a change to propagate `enable_compression` down to the transport layer and push this code into AiohttpTransport.